File: preview/hostedNumbers/authorization_document/create-default/create-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using System.Collections.Generic;
using Twilio;
using Twilio.Converters;
using Twilio.Rest.Preview.HostedNumbers;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var authorizationDocument = AuthorizationDocumentResource.Create(
            hostedNumberOrderSids: Promoter.ListOfOne("HostedNumberOrderSids"),
            addressSid: "ADaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
            email: "Email"
        );

        Console.WriteLine(authorizationDocument.Sid);
    }
}

File: preview/hosted_numbers/authorization_document/create-default/create-default.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$authorization_document = $twilio->preview->hostedNumbers->authorizationDocuments
                                                         ->create(array("hostedNumberOrderSids"), // hostedNumberOrderSids
                                                                  "ADXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", // addressSid
                                                                  "email", // email
                                                                  "contactTitle", // contactTitle
                                                                  "contactPhoneNumber" // contactPhoneNumber
                                                         );

print($authorization_document->sid);